include/lsst/jointcal/MeasuredStar.h

```diff
@@ -52,6 +52,12 @@ class MeasuredStar : public BaseStar
       { if (F)  F->MeasurementCount()++; fittedStar = F;
       }
 
+  void dump(std::ostream & stream = std::cout) const
+    {
+        BaseStar::dump(stream);
```

Comment:
Actually, since I've already cleaned up the interface some as part of this work, please feel free to suggest some improvements like that. What would you prefer instead of "dump"?